style.css

```diff
@@ -351,3 +368,172 @@ textarea::placeholder {
 input::placeholder {
   color: #979493;
 }
+
+@media screen and (min-width: 768px) {
+  header {
+    padding: 24px 150px;
+    height: 32px;
+  }
+
+  a.nav-menu {
+    display: none;
+  }
+
+  ul.nav-menu {
+    display: flex;
+    gap: 32px;
+    list-style: none;
+  }
+
+  ul.nav-menu li a {
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  #headline {
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-start;
+    gap: 87px;
+    background-image: url(assets/headline-background-desktop.png);
+    background-position: right;
+    background-repeat: no-repeat;
+    background-size: auto 100%;
+  }
+
+  footer .social-medias {
+    justify-content: center;
+  }
+
+  #headline .social-medias {
+    flex-direction: column;
+    order: 0;
+  }
+
+  #presentation-block {
+    width: 39%;
+    order: 1;
+  }
+
+  #recent-works {
+    padding: 130px 142px;
+  }
+
+  #recent-works > div:first-child {
+    display: flex;
+    flex-direction: row;
+    align-self: stretch;
+    align-items: center;
+    gap: 24px;
+    padding-bottom: 114px;
+  }
+
+  hr {
+    flex-grow: 1;
+    margin-bottom: 0;
+  }
+
+  .project-card-1 {
+    display: grid;
+    column-gap: 24px;
+    grid-template-columns: 3fr 2fr;
+    grid-template-areas:
+      "img title"
+      "img content"
```

Comment:
- [ ] [REQUIRED] Please refer to [this review](https://github.com/adebsa2401/Microverse-Portfolio/pull/5#discussion_r984017814). The project cards before hover has no button, upon hover the button appears. Kindly implement that. 
- [ ] Also, currently you have implemented a zoom-in effect which is very nice but since it is not a requirement for this project, kindly remove that effect. Part of the reasons these requirements are provided is so we learn to implement features according to specifics required in the documents from our clients. You can always add it after this project is approved.